file/src/main/scala/akka/stream/alpakka/file/impl/archive/TarArchiveEntry.scala

```diff
@@ -17,6 +17,31 @@ import akka.util.ByteString
  */
 @InternalApi private[file] object TarArchiveEntry {
 
+  /* The C structure for a Tar Entry's header is:
+   *
+   * struct header {
+   * char name[100];     // TarConstants.NAMELEN    - offset   0
```

Comment:
So, weird that this is all ascii but I've read that it was done that way by purpose to allow operators to fix archives manually...